src/overlays/actors/ovl_Obj_Dowsing/z_obj_dowsing.h

```diff
@@ -3,8 +3,18 @@
 
 #include "global.h"
 
+#define DOWSING_GET_TYPE(this) ((this)->actor.params >> 7)
+#define DOWSING_GET_FLAG(this) ((this)->actor.params & 0x7F)
```

Comment:
```suggestion
#define DOWSING_GET_TYPE(thisx) ((thisx)->params >> 7)
#define DOWSING_GET_FLAG(thisx) ((thisx)->params & 0x7F)
```
Current standard is too pass in the base actor pointer. You will have to adjust how the macros are used in `ObjDowsing_GetFlag`.